mmcv/runner/epoch_based_runner.py

```diff
@@ -17,29 +17,34 @@ class EpochBasedRunner(BaseRunner):
     This runner train models epoch by epoch.
     """
 
+    def run_iter(self, data_batch, **kwargs):
+        if self.batch_processor is not None:
+            outputs = self.batch_processor(
+                self.model, data_batch, train_mode=self.train_mode, **kwargs)
+        elif self.train_mode:
```

Comment:
I suggest making it a argument of `run_iter()` instead of an attribute of class, which remains exactly the same with previous behavior.

Reply:
https://github.com/open-mmlab/mmcv/pull/553/commits/be7444b900dc8b14db0179137358ebe0ef058bbb